Updating install instructions for B2B install in Cloud (#4364)
magento/devdocs: guides/v2.2/cloud/configure/setup-b2b.md

1. Open a terminal application.
2. Change to your local development environment root directory.

> 3. Run composer to install B2B module into your branch.

Reviewer: ```suggestion
1.  Install the B2B module using composer.
```

>         composer require magento/extension-b2b

Reviewer: ```suggestion
    ```bash
    composer require magento/extension-b2b
    ```
```

    You may be prompted to enter your Magento Authentication keys (public and private). If copying and pasting your keys, do not introduce additional spaces. Spaces could cause the following error:

        InvalidArgumentException - Could not find package magento/extension-b2b at any version for your minimum-stability (stable). Check the package spelling or your minimum-stability.

> 5. Add the updated composer.json and composer.lock file to Git:

Reviewer: ```suggestion
1.  Add and commit the updated `composer.json` and `composer.lock` files.
```

>         git add composer.json composer.lock && git commit -a -m "b2b"

Reviewer: ```suggestion
    ```bash
    git add composer.json composer.lock && git commit -a -m "install b2b module"
    ```
```
